.env.example

```diff
@@ -20,3 +20,9 @@ LADI_THRESHOLD=1.5 # float tunes the anomaly threshold based on the max value of
 
 LOCAL_DATA_FILE=validation_data/Hadoop_2k.json
 LOCAL_RESULTS_FILE=/tmp/local-results.txt
+LAD_S3_KEY=<REPLACE_WITH_S3_KEY> # The s3 key
+LAD_S3_SECRET=<REPLACE_WITH_S3_SECRET> # The s3 secret
+LAD_S3_HOST=http://localhost:8080 # The s3 endpoint url
+LAD_S3_BUCKET=<REPLACE_WITH_BUCKET_NAME> # The bucket name for s3
+LAD_MODEL_STORE=s3 # If s3 is not provided then it will write model files just to disk and not sync with s3
```

Comment:
tbh, I would prefer an `.env.example` that just works if copied to `.env`. How about making the default empty and the above too so that the upload to S3 doesnt happen?

Reply:
Good point. I'll leave it blank.